recommendation/app.py

```diff
@@ -0,0 +1,59 @@
+from flask import Flask, json
+import model.user_rec as userRec
+import model.post_rec as postRec
+
+import utils.backend as dUsers
+
+import json
+
+import os
+from dotenv import load_dotenv
+load_dotenv()
+
+from flask_cors import CORS, cross_origin
+
+
+app = Flask(__name__)
+cors = CORS(app)
+app.config['CORS_HEADERS'] = 'Content-Type'
+
+def getRecommendations(user_id, all_users_data, liked_matrix, post_matrix):
```

Comment:
nit: `get_recommendations` + generally snake case in the rest of the file, including the `userRec` and `postRec` imports!